### PR TITLE
Add test for Configuration as Code support

### PR DESCRIPTION
**Pull request by rsandell**

### CODE REVIEW

**Comment by rsandell** [comment]
@oleg-nenashev @jglick I see you are listed as maintainers. Could I get a quick review please?

File: src/test/java/hudson/os/windows/LauncherCasCRoundTripTest.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        assertEquals("-Xmx200", winLauncher.vmargs);
+        final ManagedWindowsServiceAccount account = winLauncher.getAccount();
+        assertThat(account, instanceOf(ManagedWindowsServiceAccount.AnotherUser.class));
+        ManagedWindowsServiceAccount.AnotherUser aua = (ManagedWindowsServiceAccount.AnotherUser) account;
+        assertEquals("bob", aua.userName);
+    }
+
+    @Override
+    protected String stringInLogExpected() {
+        return "Windows";
```
Comment (varyvol) [inline src/test/java/hudson/os/windows/LauncherCasCRoundTripTest.java:40]: Isn't there something a little more specific?

**Review by oleg-nenashev — APPROVED** [review]
Thanks! I think it is a good addition to the plugin.

**Review by MRamonLeon — APPROVED** [review]

File: src/test/java/hudson/os/windows/LauncherCasCRoundTripTest.java (hunk 1 above)
Comment (rsandell) [reply src/test/java/hudson/os/windows/LauncherCasCRoundTripTest.java:40]: Maybe, I'm just having a hard time understanding the need for this check, either it's configured or not :)

**Review by alecharp — APPROVED** [review]

**Comment by rsandell** [comment]
@oleg-nenashev We have three approvals, can you merge please?

**Comment by oleg-nenashev** [comment]
I will merge it, but I do not plan to release it in short term. If there are any features/fixes contributed, I will ship this change with them. Thanks for your contribution!

**Comment by rsandell** [comment]
Thank you @oleg-nenashev! I don't see a need to release it, it's only a test to make sure future releases doesn't break jcasc :)

**Pull request merged**